<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
font: 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>321</width>
      <height>591</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="25,0,15,60">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,9">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>227</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>30</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 30pt &quot;Bahnschrift&quot;;
color: rgb(82,201,220);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>227</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>30</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="toolTipDuration">
          <number>5</number>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 30pt &quot;Bahnschrift&quot;;
color: rgb(36,36,36);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="toolTipDuration">
        <number>1</number>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>37</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;Bahnschrift&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>15</number>
       </property>
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(36, 36, 36);
	background-color: rgb(242, 248, 252);
	
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_6">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(36, 36, 36);
	background-color: rgb(242, 248, 252);
	
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>6</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(36, 36, 36);
	background-color: rgb(242, 248, 252);
	
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_7">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(36, 36, 36);
	background-color: rgb(242, 248, 252);
	
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>7</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_8">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(36, 36, 36);
	background-color: rgb(242, 248, 252);
	
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(36, 36, 36);
	background-color: rgb(242, 248, 252);
	
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="pushButton_minus">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(255, 255, 255);
	background-color: rgb(82,201,220);
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_percent">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(255, 255, 255);
	background-color: rgb(82,201,220);
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>%</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_dot">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(36, 36, 36);
	background-color: rgb(242, 248, 252);
	
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>.</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="3" rowspan="2">
        <widget class="QPushButton" name="pushButton_plus">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(255, 255, 255);
	background-color: rgb(82,201,220);
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_0">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(36, 36, 36);
	background-color: rgb(242, 248, 252);
	
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_plusMinus">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(255, 255, 255);
	background-color: rgb(82,201,220);
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>+/-</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(36, 36, 36);
	background-color: rgb(242, 248, 252);
	
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="pushButton_isEqualTo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(255, 255, 255);
	background-color: rgb(82,201,220);
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>=</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushButton_multiply">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(255, 255, 255);
	background-color: rgb(82,201,220);
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>*</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_5">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(36, 36, 36);
	background-color: rgb(242, 248, 252);
	
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_9">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(36, 36, 36);
	background-color: rgb(242, 248, 252);
	
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>9</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_divide">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(255, 255, 255);
	background-color: rgb(82,201,220);
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>/</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_CE">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border: none;
color:rgb(255, 255, 255);
	background-color: rgb(82,201,220);
	font: 20pt &quot;Bahnschrift&quot;;
border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>CE</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton_1</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_8</tabstop>
  <tabstop>pushButton_9</tabstop>
  <tabstop>pushButton_0</tabstop>
  <tabstop>pushButton_dot</tabstop>
  <tabstop>pushButton_isEqualTo</tabstop>
  <tabstop>pushButton_CE</tabstop>
  <tabstop>pushButton_plusMinus</tabstop>
  <tabstop>pushButton_percent</tabstop>
  <tabstop>pushButton_divide</tabstop>
  <tabstop>pushButton_multiply</tabstop>
  <tabstop>pushButton_minus</tabstop>
  <tabstop>pushButton_plus</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
